ansible/app_options_files/public_testnet

```diff
@@ -22,15 +22,51 @@
         },
```

Comment:
should we have all this files in the official playbook?

Reply:
This was separated so the team can update it without going into the ansible details. It never really took off, so we could migrate it into the playbook. Seems like a low priority though, I'd put it off for now. It's not fun to troubleshoot automation because of a refactor.